app/helpers/application_helper.rb

```diff
@@ -57,4 +57,20 @@ def pdf_stylesheet_pack_tag(source)
       wicked_pdf_stylesheet_pack_tag(source)
     end
   end
+
+  def line_items_for_invoice(order)
```

Comment:
This looks good the ApplicationHelper is for code that's used in most parts of the application. This code is only used on invoices. I actually like your idea of sorting within the model.